src/actions/accountActions.js

```diff
@@ -76,4 +77,37 @@ function decryptCloudWalletAccount (accountData: AccountData, password: string)
   }
 }
 
-export { syncWithNetwork, getTxFee, decryptCloudWalletAccount }
+async function _verifyPassword (transferInfo: {
+  transferId: ?string,
+  account: AccountData,
+  password: string
+}) {
+  let { transferId, account, password } = transferInfo
+
+  if (transferId) {
+    // retrieve password from drive
+    let transferData = await getTransferData(transferId)
+    password = Base64.decode(transferData.password)
+  }
+
+  let _account = createAccount(account)
+  await _account.decryptAccount(password)
+  return _account.getAccountData()
+}
+
+function verifyPassword (transferInfo: {
```

Comment:
I suggest changing the names of these functions to verifyEscrowAccountPassword. Cloud wallet has its own decrypt action function, and now verifyPassword is just for escrow, so better to name theses function more precisely.